feature-libs/product-configurator/rulebased/components/conflict-solver-dialog/index.ts

```diff
@@ -0,0 +1,8 @@
+/*
+ * SPDX-FileCopyrightText: 2022 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+export * from './configurator-conflict-solver-dialog.component';
```

Comment:
Please also export the event listener (or whatever we call this class)

Reply:
Done!